lib/intercom/service/conversation.rb

```diff
@@ -52,6 +52,13 @@ def assign(reply_data)
         assignee_id = reply_data.fetch(:assignee_id) { fail 'assignee_id is required' }
         reply reply_data.merge(message_type: 'assignment', assignee_id: assignee_id, type: 'admin')
       end
+
+      def run_assignment_rules(id)
```

Comment:
Looks good! It'd be cool to get a test for this behaviour as well? 😄

Reply:
😲We already pay you a lot for you product man.
Recently we had to debug your api that was sending wrong data on the feedback webhook.

We can code for you as well, but maybe you could thanks us with a free month then 😉

Reply:
Unfortunately that's not up to me 😄 the SDKs are a community driven project.

>Recently we had to debug your api that was sending wrong data on the feedback webhook.

Have you reached out to support around this?